src/Symfony/Bridge/Twig/Mime/NotificationEmail.php

```diff
@@ -82,9 +80,7 @@ public function markAsPublic(): static
         return $this;
     }
 
-    /**
-     * @return $this
```

Comment:
this must be kept as it is more specific than `static`. This is true for all methods using it.